### ISSUE TITLE
We are looking for some help to improve readme file, from spanish to english.

### ISSUE THREAD

**Issue by xaca** [xaca/java_fundamentals#5]
We like to improve this course, is free, is about java basics. For a CS1 course.

**Comment by AlexanderDavid** [comment]
Are you interested in changing the language within the code too? Because that is something I can help with as well. What would you like the readme file to say? Just a basic this is java, how to download and set up an IDE?

**Comment by lucis** [comment]
#7

**Comment by xaca** [comment]
@AlexanderDavid yes maybe a getting started instruction, we have a step by step video, but would be nice a better readme file.

**Comment by AlexanderDavid** [comment]
@xaca I'll tackle the getting started in the readme. Using windows 10 and Eclipse

**Comment by xaca** [comment]
@AlexanderDavid Good that sounds great, thank you

**Comment by f4ww4z** [comment]
@xaca May I also help with the readme? I want to improve the spelling and grammar in the readme.

**Comment by xaca** [comment]
@f4ww4z Ok, we need that, go ahead!
